my-react-app/src/Components/Inputs/Inputs.js

```diff
@@ -6,17 +6,23 @@ import InputLabel from "@material-ui/core/InputLabel";
 import MenuItem from "@material-ui/core/MenuItem";
 import FormControl from "@material-ui/core/FormControl";
 import Select from "@material-ui/core/Select";
+import "./Inputs.scss";
 
 const Inputs = ({ setNote }) => {
+  let today = new Date();
+  let day = `0${today.getDate()}`.slice(-2);
```

Comment:
0?

Reply:
yep, because we need date format 06.16.2021. New Date creates date in format 6.16.2021. And i dont know another way how to get it